lib/elixir/lib/kernel.ex

```diff
@@ -261,6 +261,8 @@ defmodule Kernel do
   @doc """
   Returns the head of a list. Raises `ArgumentError` if the list is empty.
 
+  It works with improper lists.
```

Comment:
@michalmuskala that's exactly the reason why I think it is worth mentioning it. Because this is one of the few that deal with improper lists.